pkg/controller/servicemesh/controlplane/addons.go

```diff
@@ -114,6 +114,20 @@ func (r *controlPlaneInstanceReconciler) patchKiali(ctx context.Context, grafana
 		}
 	}
 
+	// istio
```

Comment:
Just wondering if this could be covered in tests

Reply:
It would be very hard, and it's much easier to cover this case in the existing tests in maistra-test-tool.